traitsui/testing/tester/ui_tester.py

```diff
@@ -24,108 +24,7 @@
-
-    Locating GUI elements
-    ---------------------
-    After creating an ``UI`` object, ``UITester.find_by_name`` can be used
-    to locate a specific UI target::
-
-        obj = App()
-        tester = UITester()
-        with tester.create_ui(obj) as ui:
-            wrapper = tester.find_by_name(ui, "text")
-
-    The returned value is an instance of ``UIWrapper``. It wraps the
-    UI target instance found and allows further test actions to be performed on
-    the target.
-
-    Performing an interaction (commands)
-    ------------------------------------
-    After locating the GUI element, we typically want to perform some user
-    actions on it for testing. Examples of user interactions that produce side
-    effects are clicking or double clicking a mouse button, typing some keys
-    on the keyboard, etc.
-
-    To perform an interaction for side effects, call the ``UIWrapper.perform``
-    method with the interaction required. A set of predefined command types can
-    be found in the ``traitsui.testing.tester.command``.
-
-    Example::
-
-        with tester.create_ui(app, dict(view=view)) as ui:
-            tester.find_by_name(ui, "button").perform(command.MouseClick())
-            assert app.clicked
-
-    Inspecting GUI element (queries)
-    --------------------------------
-    Sometimes, the test may want to inspect visual elements on the GUI, e.g.
-    checking if a textbox has displayed the expected value.
-
-    To perform an interaction for queries, call the ``UIWrapper.inspect``
-    method with the query required. A set of predefined query types can be
-    found in the ``traitsui.testing.tester.query``.
-
-    Example::
-
-        with tester.create_ui(app, dict(view=view)) as ui:
-            text = (
-                tester.find_by_name(ui, "text").inspect(query.DisplayedText())
-            )
-            assert text == "Hello"
-
-    Extending the API
-    -----------------
-    The API can be extended by defining a registry for mapping target and
-    interaction types to a specific implementation that handles the
-    interaction.
-
-    For example, suppose there is a custom UI target ``MyEditor``, to implement
-    a custom interaction type ``ManyMouseClick`` for this target::
-
-        custom_registry = TargetRegistry()
-        custom_registry.register_handler(
-            target_class=MyEditor,
-            interaction_class=ManyMouseClick,
-            handler=lambda wrapper, interaction: wrapper._target.do_something()
-        )
-
-    Then the registry can be used in a UITester::
-
-        tester = UITester(registries=[custom_registry])
-
-    This is how TraitsUI supplies testing support for specific editors; the
-    default setup of ``UITester`` comes with a registry for testing TraitsUI
-    editors.
-
-    Similar the location resolution logic can be extended.
-
-    See documentation on ``TargetRegistry`` for details.
+    See :ref:`testing` in the User Manual for further details.
```

Comment:
This forces devs to look at docs instead of just falling back to always look at source so 👍

Reply:
Nit : Can we use "See :ref:`testing` Section of the ..." to ensure that devs don't just go searching for `testing` in the user docs instinctively instead of just browsing and finding the actual "testing" section?